ion/timestamp.go

```diff
@@ -482,8 +470,23 @@ func (ts Timestamp) String() string {
 		// This position better be right of 'T'
 		if index != -1 && tIndex < index {
 			zeros := strings.Builder{}
-			zeros.WriteByte('.')
-			for i := uint8(0); i < ts.numFractionalSeconds; i++ {
+			numTrailingZeros := 0
```

Comment:
You should make the same variable renaming improvements done in https://github.com/justing-bq/ion-go/pull/1/commits/e439622f2954e0dccf1f3eebe3e5d0cb116154ae

Reply:
Renamed the variables